Block/Adminhtml/TransactionSync.php

```diff
@@ -29,38 +29,25 @@ class TransactionSync extends PopupField
     /**
      * @var string
      */
-    // @codingStandardsIgnoreStart
     protected $_template = 'Taxjar_SalesTax::transaction_sync.phtml';
-    // @codingStandardsIgnoreEnd
-
-    /**
-     * @var \Magento\Backend\Model\UrlInterface
-     */
-    protected $backendUrl;
-
-    /**
-     * @var \Magento\Framework\App\Config\ScopeConfigInterface
-     */
-    protected $scopeConfig;
 
     /**
      * @var \Taxjar\SalesTax\Helper\Data
      */
     protected $helper;
 
     /**
+     * @param TaxjarHelper $helper
      * @param Context $context
      * @param UrlInterface $backendUrl
      * @param array $data
      */
     public function __construct(
+        TaxjarHelper $helper,
         Context $context,
         UrlInterface $backendUrl,
-        TaxjarHelper $helper,
         array $data = []
     ) {
-        $this->scopeConfig = $context->getScopeConfig();
```

Comment:
These scopeConfig and backendUrl properties are already being set in the parent constructor.